rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/util/LocalJobFriendlyPackerCommandFactory.groovy

```diff
@@ -26,7 +26,18 @@ class LocalJobFriendlyPackerCommandFactory implements PackerCommandFactory {
     def packerCommand = [baseCommand, "packer", "build", "-color=false"]
     parameterMap.each { key, value ->
       if (key && value) {
-        def keyValuePair = value instanceof String && value.contains(" ") ? "$key=\"$value\"" : "$key=$value"
+        def cleansedValue = value instanceof String ? value.trim() : value
```

Comment:
I can confirm the change on line 29 being ok. We actually did this change locally a while a go, but never got around to push it upstream. We've also stopped removing leading/trailing space in the `install_packages.sh` script, so I'm curious as to why it is needed here? And if it's needed I think that `trim()` would handle this case, right?